src/components/--Hero/Hero.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styled from "styled-components";
+const defaultCover = require("./../../images/jpg/cover-default.jpg");
+
+const HeroHeader = styled.header`
+  background-image: url(${(props) => props.imageUrl || defaultCover});
+  background-size: cover;
+  height: 480px;
+  width: 100%;
+  position: relative;
+  &::after {
+    content: "";
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    background: ${(props) =>
+      props.hasOverlay &&
```

Comment:
Conditional overlay, because we might want to reuse Hero without overlay at pages